src/utils/index.ts

```diff
@@ -1,12 +1,12 @@
 import { format, hoursToMilliseconds, millisecondsToHours } from "date-fns";
 import { NextApiRequest } from "next";
 import slugify from "slugify";
-import { MetadataProps } from "../../types";
+import { Flatten, MetadataProps } from "../../types";
 import ClockIcon from "../components/svgs/ClockIcon";
 import GithubIcon from "../components/svgs/GithubIcon";
 import LaptopIcon from "../components/svgs/LaptopIcon";
 import config from "../config/config.json";
-
+import yaml from "js-yaml";
```

Comment:
there should be an empty line between the imports and the rest of the code.
also, based on the rest of the imports here, libraries are imported first.